### DISCUSSION TITLE
Loop video

### DISCUSSION THREAD

**Discussion by Otelow** [WarmUpTill/SceneSwitcher#1197] [Q&A]
Hello, I would like to know if it is possible to loop using the plugin. I would like my media source to restart every X minutes, is that possible?

**Comment by WarmUpTill** [comment]
Yes, that should be possible using a macro like this:

![image](https://github.com/user-attachments/assets/00655253-ed50-4e85-9a78-3cb05b8ffc65)

Let me know if you have any questions!
